### DISCUSSION TITLE
Student Account not proved

### DISCUSSION THREAD

**Discussion by Tndot** [community/community#51629] [GitHub Education]
I am trying to create my student account but always giving this error:

The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.

You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

I applied in the school teaching area, but still prompted this error。

**Comment by sameerch4n** [comment]
Hello @Tndot,
I understand that you are facing an issue with accessing the GitHub Student Developer Pack, similar to what I faced as well while trying to register. It appears that your school may not have registered with GitHub to offer student subscriptions or that your program may not be eligible for a student account if it is remote. I suggest seeking support from the technical department to assist you with obtaining a student subscription or account.

**Reply by Tndot** [reply]
Hello @samunicode,  
Thank you for your suggestion. How to seek support from the technical department? I previously asked GitHub support about this issue, but they directed me to direct my inquiries to the GitHub community.

**Comment by samus-aran** [comment]
Hi @Tndot 👋 

It's stating that you are far from Campus, what campus did you select on the map when asked for the school you're attending? 

When asked, please enable the application to discover your location from the browser pop-up. Since settings for every browser are different, if you are having difficulty enabling us to see your location you should seek assistance from your browser's support pages.

Also, ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.
